### ISSUE TITLE
LSTM always predict zeros

### ISSUE THREAD

**Issue by sun-peach** [keras-team/keras#9338]
Hi,

I am building up a network with maxout layers and LSTM layers. The architecture is shown below:
![image](https://user-images.githubusercontent.com/14579257/35983211-2d80a626-0cb7-11e8-806b-ea4e2286055a.png)

This is part of a network. I am trying to extract hidden value from LSTM_2 output given the input_1. When I do prediction and extract hidden values, it is always zero. The activation function is tanh. The whole network works fine (at least, the performance is good). I did not use batch norm. I use Adam to train and the dropout rate is about 0.1. I checked the hidden values from dropout_2, they have numbers. From lstm_1, the prediction is zero also. It seems when it goes through a lstm layer, it turns to 0.

I am using keras 1.1.0 (I did not use the latest version, because some of my packages have conflict with the keras 2.0+ versions).

Can anyone tell me why it happens?

**Comment by brainnoise** [comment]
Can you please share standalone code snippet?
